package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ask-cli-x",
-  "version": "0.1.1",
+  "version": "0.1.0",
```

Comment:
This shouldn't be bumped down?

Reply:
This is because we were planning for a release for [commit](https://github.com/alexa-labs/ask-cli/commit/4665bde33fa2c6f7e10dabd46257daef5d2aadba) which didn't happen back then. And we will be using standard-version to bump and modify changelog, thus I revert the version and delete the changelog in this commit.